homeassistant/components/template/template_entity.py

```diff
@@ -285,13 +286,13 @@ def __init__(
             ]
             | None
         ) = None
+        self._run_variables: ScriptVariables | dict = {}
```

Comment:
You could make this just a type hint to avoid the unnecessary initialization:
```suggestion
        self._run_variables: ScriptVariables | dict
```